envs/ci/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+RUN apt-get update \
+    # we need 'enchant-2' for pylint's spellchecker
+    && apt-get -y install enchant-2 \
+    && pip install --upgrade pip \
+    && pip install poetry==1.5.1
+
+RUN poetry config virtualenvs.create false
+
+COPY poetry.lock pyproject.toml ./
+
+RUN poetry lock --check
+
+RUN poetry install --with=lint,test
```

Comment:
According to our last discussion, `test` dependencies will be added later, when we will setup CI for tests as a part of the same story (https://github.com/fenya123/bingin/issues/11)